index.js

```diff
@@ -3,3 +3,43 @@ const modal = document.querySelector('.modal');
 const wrapper = document.querySelector('.wrapper1');
 const modalX = document.querySelector('.close');
 const links = document.querySelectorAll('.modal-link');
+
+function modalHide() {
+  modal.style.display = 'none';
+  wrapper.style.filter = 'none';
+}
+
+function windowClickHandler(event) {
+  if ((event.target === modal) || (event.target === links)) {
+    modalHide();
+  }
+}
+
+function modalShow() {
+  modal.style.display = 'flex';
+  wrapper.style.filter = 'blur(0.4rem)';
+  modal.scrollIntoView({ behavior: 'smooth' });
+}
```

Comment:
- [ ] Kindly consider making the modal take the height of the screen by changing the height of the modal to `100vh` at [this line](https://github.com/Bestbynature/portfolio/blob/15a22bc86d493c35a0523641962692d8678559bd/styles.css#L87)
- [ ] Also consider sticking to the design by using the correct font size and also make sure to put the close icon in the proper position like the Figma design to make your design identical to Figma

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/223209571-46787eaf-3621-4b73-9bbc-30173a1b18b5.png) | ![image](https://user-images.githubusercontent.com/53402600/223210388-c93c741b-6bdc-4fca-ab81-829a6660a039.png) |